<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkXnatUploadFromDataStorageDialog</class>
 <widget class="QDialog" name="QmitkXnatUploadFromDataStorageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblSelectData">
     <property name="text">
      <string>Select the data for upload:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkDataStorageComboBox" name="cBDataSelection"/>
   </item>
   <item>
    <widget class="QCheckBox" name="cbUploadMITKProject">
     <property name="toolTip">
      <string>Uploads all data as MITK project file</string>
     </property>
     <property name="text">
      <string>Upload scene as MITK project file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leMITKProjectFileName">
     <property name="placeholderText">
      <string>Enter project file name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpload">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
